src/jarabe/main.py

```diff
@@ -58,7 +58,6 @@
 gi.require_version('Wnck', '3.0')
 gi.require_version('SugarExt', '1.0')
 gi.require_version('GdkX11', '3.0')
-gi.require_version('WebKit', '3.0')
```

Comment:
In this patch you use Webkit 4.0, as seen in one of my comments I did not had 4.0 installed, should we add a try-except for this? I think this affects sugar-build with prebuilt image as well